app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,23 @@
+
+<div class='container' >
+  <div class='loginForm'>
+      <h1>Recipes Lists</h1>
+      <div class='food-add-button'>
+        <%= button_to link_to 'Add Recipe', new_user_recipe_path %>
+      </div>
+
+      <ul class="recipes-list">
+        <% @recipes.each do |recipe| %>
+          <li>
+              <div class="label">
+                  <b><%= link_to recipe.name , user_recipe_path(@user, recipe) %></b>
+                  <%= button_to 'Remove', user_recipe_path(@user, recipe), method: :delete, class: 'remove-recipe-button'%>
```

Comment:
- [ ] Same with recipes. I can also delete recipes from the lists created by different users, but owners should only do it.

Reply:
@Deepakdanger, same here, It was a missed, and now fixing is done.